manifests/server/startup.pp

```diff
@@ -0,0 +1,90 @@
+# == Define: dns::server::startup
```

Comment:
I think we are ready to need this. I don't really like the name "startup" though, it makes me think it handles init script.

How about class: dns::server::defaults_file ?